Project.toml

```diff
@@ -0,0 +1,25 @@
+name = "BinDeps"
+uuid = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
+version = "0.8.10"
+
+[deps]
+BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
```

Comment:
thats new

Reply:
It doesn't seem to be used?

Reply:
I was getting an error without this:
```
ERROR: LoadError: LoadError: ArgumentError: Package BinaryProvider not found in current path:
- Run `import Pkg; Pkg.add("BinaryProvider")` to install the BinaryProvider package.

Stacktrace:
 [1] require(::Module, ::Symbol) at ./loading.jl:876
 [2] include at ./boot.jl:328 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1094
 [4] include at ./Base.jl:31 [inlined]
 [5] eval_anon_module(::BinDeps.PackageContext, ::String) at /home/mose/.julia/dev/BinDeps/src/BinDeps.jl:557
 [6] debug_context(::String) at /home/mose/.julia/dev/BinDeps/src/debug.jl:52
 [7] debug(::Base.TTY, ::String) at /home/mose/.julia/dev/BinDeps/src/debug.jl:57
 [8] debug(::String) at /home/mose/.julia/dev/BinDeps/src/debug.jl:69
 [9] top-level scope at /home/mose/.julia/dev/BinDeps/test/runtests.jl:23
 [10] include at ./boot.jl:328 [inlined]
 [11] include_relative(::Module, ::String) at ./loading.jl:1094
 [12] include(::Module, ::String) at ./Base.jl:31
 [13] include(::String) at ./client.jl:431
 [14] top-level scope at none:5
in expression starting at /home/mose/.julia/packages/Cairo/ZbzIH/deps/build.jl:1
in expression starting at /home/mose/.julia/dev/BinDeps/test/runtests.jl:23
ERROR: Package BinDeps errored during testing
```
Maybe it's only used in tests?

Reply:
It was needed only by (new) Cairo during test.  But we need old Cairo for testing.  Thanks for pointing this out